config/default/user.role.agency_administrator.yml

```diff
@@ -0,0 +1,43 @@
+uuid: a352e518-129e-47f3-99bb-3b6879a5254e
+langcode: en
+status: true
+dependencies:
+  module:
+    - lightning_core
+third_party_settings:
+  lightning_core:
+    description: ''
+id: agency_administrator
+label: 'Agency Administrator'
+weight: 11
+is_admin: null
+permissions:
+  - 'access content overview'
+  - 'access devel information'
```

Comment:
I don't think we would want them to have access to `devel` info. Rather than removing this, though, maybe you can just disable `devel` as part of this ticket.